content/en/tracing/profiler/compare_profiles.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Compare Profiles
+kind: documentation
+further_reading:
+    - link: 'tracing/profiler/enabling'
+      tag: 'Documentation'
+      text: 'Enable continuous profiler for your application'
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tag: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog'
+---
+
+Continuous Profiler can compare two profiles or profile aggregations with each other to help you identify code performance improvements, regressions, and structural changes. You can compare a profile with:
+
+- The same profile at a different time,
+- An average of the service's profiles over time, or
+- A profile with a different set of Datadog tags (for example, environment, version, or data center). 
```

Comment:
Maybe we should say Profiles since we are aggregating a bunch of 60s profiles? So it reads... "Profiles with a different set of...."

Reply:
```suggestion
- Profiles with a different set of Datadog tags (for example, environment, version, or data center). 
```